### DISCUSSION TITLE
Problema para utilizar Copilot

### DISCUSSION THREAD

**Discussion by dannyjimenez110** [community/community#147450] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Trilogy::ProtocolError: 1062: Duplicate entry 'xxxxxxx' for key 'copilot_limited_users.index_copilot_limited_users_on_user_id' (trilogy_query_recv)

**Comment by Akash1134** [comment, marked as answer]
Hi @dannyjimenez110 , This issue affecting the complimentary access to GitHub Copilot Pro for students, teachers and maintainers has been resolved! Continue enjoying your complimentary access to GitHub Copilot Pro!
